test/parallel/test-assert.js

```diff
@@ -406,7 +406,7 @@ assert.doesNotThrow(function() { assert.ifError(); });
 
 assert.throws(() => {
   assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
-}, /Got unwanted exception: user message/,
+}, /Got unwanted exception:\ntest: user message/,
               'a.doesNotThrow ignores user message');
```

Comment:
@joyeecheung completely agree with your here. I added another test case that tests unwanted exception message. I hope it is ok.